how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -109,20 +118,30 @@ Environment="HTTPS_PROXY=${https_proxy}"
 Environment="NO_PROXY=${no_proxy}"
 EOT
 done
+```
+```bash
 $ sudo systemctl daemon-reload
 ```
 
-###  Start Kubernetes with kubeadm 
+###  Start Kubernetes with `kubeadm`
+
+
+- Make sure containerd is up and running
 
 ```bash
-# Mark sure containerd is up and running
 $ sudo systemctl restart containerd
 $ sudo systemctl status containerd
+```
+
+- Prevent conflicts of docker iptables rules & k8s pod communication
 
-# Prevent docker iptables rules conflict with k8s pod communication
+```bash
 $ sudo iptables -P FORWARD ACCEPT
```

Comment:
I'm an `iptables` numpty - is this basically disabling a bunch of iptables functionality globally? If so, maybe we want to put some sort of note/warning in here to tell users this is a dev/test measure, and for any sort of deployment they might want to do something more 'robust' ?